<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SmartCalc</class>
 <widget class="SmartCalc" name="SmartCalc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>701</width>
    <height>471</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>371</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="SmartCalc" stdset="0">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>701</width>
     <height>461</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <widget class="QLineEdit" name="lineEdit_expression">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>701</width>
      <height>41</height>
     </rect>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhNone</set>
    </property>
    <property name="text">
     <string>1+1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_D_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_answer">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>70</y>
      <width>281</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_D_8">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>270</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_D_9">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>270</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_M_2sqrt">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>170</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>√</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_M_div">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>420</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_D_6">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>320</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_M_mul">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>370</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_M_pow">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>170</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Xⁿ</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_D_5">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>320</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_D_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_D_3">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>370</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_M_add">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>270</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_M_sub">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>320</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_D_0">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>420</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_D_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>370</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_M_mod">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>mod</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_D_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_bracket_c">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>220</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_constant_e">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>220</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>e</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_bracket_o">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>(</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_constant_pi">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>220</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>pi</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_E_clean">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>170</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>C</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_E_calc">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>70</y>
      <width>111</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_D_dot">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>420</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_V_graph">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>120</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Graph</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_X_value">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>136</y>
      <width>111</width>
      <height>24</height>
     </rect>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhPreferNumbers</set>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_text_scale">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>120</y>
      <width>21</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>🔍</string>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_text_X">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>113</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QLabel" name="lineEdit_infotext">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>601</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>HELLO this is text you can copy</string>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_lifetime_calc">
    <property name="geometry">
     <rect>
      <x>605</x>
      <y>0</y>
      <width>92</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="text">
     <string>Lifetime calc</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_M_log">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>120</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>log10</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_M_atan">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>120</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>atan</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_M_asin">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>asin</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_M_cos">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>70</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>cos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_M_acos">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>120</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>acos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_M_ln">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>70</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>ln</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_M_tan">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>70</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>tan</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_M_sin">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>sin</string>
    </property>
   </widget>
   <widget class="QCustomPlot" name="grid" native="true">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>170</y>
      <width>401</width>
      <height>291</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_V_graph_2">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>120</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Ещё→</string>
    </property>
   </widget>
   <zorder>pushButton_D_7</zorder>
   <zorder>lineEdit_answer</zorder>
   <zorder>pushButton_D_8</zorder>
   <zorder>pushButton_D_9</zorder>
   <zorder>pushButton_M_2sqrt</zorder>
   <zorder>pushButton_M_div</zorder>
   <zorder>pushButton_D_6</zorder>
   <zorder>pushButton_M_mul</zorder>
   <zorder>pushButton_M_pow</zorder>
   <zorder>pushButton_D_5</zorder>
   <zorder>pushButton_D_4</zorder>
   <zorder>pushButton_D_3</zorder>
   <zorder>pushButton_M_add</zorder>
   <zorder>pushButton_M_sub</zorder>
   <zorder>pushButton_D_0</zorder>
   <zorder>pushButton_D_2</zorder>
   <zorder>pushButton_M_mod</zorder>
   <zorder>pushButton_D_1</zorder>
   <zorder>pushButton_bracket_c</zorder>
   <zorder>pushButton_constant_e</zorder>
   <zorder>pushButton_bracket_o</zorder>
   <zorder>pushButton_constant_pi</zorder>
   <zorder>pushButton_E_clean</zorder>
   <zorder>pushButton_E_calc</zorder>
   <zorder>pushButton_D_dot</zorder>
   <zorder>pushButton_V_graph</zorder>
   <zorder>lineEdit_X_value</zorder>
   <zorder>lineEdit_text_scale</zorder>
   <zorder>label_text_X</zorder>
   <zorder>lineEdit_infotext</zorder>
   <zorder>checkBox_lifetime_calc</zorder>
   <zorder>pushButton_M_log</zorder>
   <zorder>pushButton_M_atan</zorder>
   <zorder>pushButton_M_asin</zorder>
   <zorder>pushButton_M_cos</zorder>
   <zorder>pushButton_M_acos</zorder>
   <zorder>pushButton_M_ln</zorder>
   <zorder>pushButton_M_tan</zorder>
   <zorder>pushButton_M_sin</zorder>
   <zorder>lineEdit_expression</zorder>
   <zorder>grid</zorder>
   <zorder>pushButton_V_graph_2</zorder>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_credit_summ_1">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>20</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="decimals">
    <number>3</number>
   </property>
   <property name="minimum">
    <double>1.000000000000000</double>
   </property>
   <property name="maximum">
    <double>100000000.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_text_X_2">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>0</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>кредитный</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_text_X_3">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>20</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>сумма кредита</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_text_X_4">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>40</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>срок (мес)</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_credit_range_2">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>40</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>1000</number>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_paym_3">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>60</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="decimals">
    <number>3</number>
   </property>
   <property name="minimum">
    <double>0.001000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_text_X_5">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>60</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>ставка</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QCheckBox" name="credit_check_box_1">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>80</y>
     <width>131</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="text">
    <string>аннуитетный</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="credit_check_box_2">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>80</y>
     <width>201</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>дифференцированный</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_text_X_6">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>20</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>ежемесячно</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="credit_text_1">
   <property name="geometry">
    <rect>
     <x>1060</x>
     <y>20</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhPreferNumbers</set>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="credit_text_2">
   <property name="geometry">
    <rect>
     <x>1060</x>
     <y>40</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhPreferNumbers</set>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="credit_text_3">
   <property name="geometry">
    <rect>
     <x>1060</x>
     <y>60</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhPreferNumbers</set>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_text_X_7">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>40</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>переплата</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_text_X_8">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>60</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>общая выплата</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SmartCalc</class>
   <extends>QWidget</extends>
   <header>smartcalc.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
   <slots>
    <slot>areachange()</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
